### ISSUE TITLE
About 01-AMASS_Visualization

### ISSUE THREAD

**Issue by cotton-ahn** [nghorbani/amass#31]
Dear authors,

thanks for releasing this awesome code for using AMASS dataset! 

I found a very small bug: in the 01-AMASS_Visualization.ipynb, the "def vis_body_joints" does not work when using the very recent body_visualizer package.

    joints_mesh = points_to_spheres(joints, vc = colors['red'], radius=0.005)

should be changed into 

    joints_mesh = points_to_spheres(joints, point_color = colors['red'], radius=0.005)

I also find this kind of naming issue in body_visualizer happens when using vposer's(human_body_prior) ik_engine.py and ik_example_joints.py.

Thanks a lot for this repository and have a nice day :)

**Comment by cotton-ahn** [comment]
I checked that maintainers have fixed this issue. But it would have been also better if they leave some comments in here :/ but anyway I am closing this issue.

**Comment by cotton-ahn** [comment]
I've found out another bug. Due to changes in body model prior package, :

bm = BodyModel(bm_fname=bm_fname, num_betas=num_betas, num_dmpls=num_dmpls, dmpl_fname=dmpl_fname).to(comp_device)

--> bm_fname should be bm_path. If not, below error occurs

TypeError: __init__() got an unexpected keyword argument 'bm_fname'

also same to the dmpl_fname, which should be path_dmpl.

and model_type should be defined as model_type = 'smplh'

so the final line should be

`bm = BodyModel(bm_path=bm_path, num_betas=num_betas, model_type='smplh', num_dmpls=num_dmpls, path_dmpl = path_dmpl).to(comp_device)`

but if I do not specify the batch_size, it does not work.

TLDR : it seems this repository does not work anymore when used with recent body_model_prior. -- repo needs to be updated..../

**Comment by Recialhot** [comment]
Hello, I would like to ask what coordinate system AMASS uses and including its visualization?

**Comment by nghorbani** [comment]
Fixed by 36d86b5 on master: requirements.txt and setup.py pin human_body_prior to a commit whose BodyModel takes bm_fname, num_betas, num_dmpls and dmpl_fname exactly as the notebooks call it; the TypeError came from older human_body_prior installs (the PyPI release or the v1 bm_path API). Reinstall with pip install -r requirements.txt.
